src/main/java/com/spotify/reaper/service/SegmentRunner.java

```diff
@@ -318,6 +333,21 @@ boolean canRepair(RepairSegment segment, String keyspace, JmxProxy coordinator,
     return true;
   }
 
+  private boolean IsRepairRunningOnOneNode(RepairSegment segment) {
+    for(RepairSegment segmentInRun:context.storage.getRepairSegmentsForRun(segment.getRunId())){
+      try (JmxProxy hostProxy = context.jmxConnectionFactory.connect(segmentInRun.getCoordinatorHost())) {
+        if(hostProxy.isRepairRunning()) {
+          return true;
+        }
+      } catch (ReaperException e) {
+        LOG.error("Unreachable node when trying to determine if repair is running on a node. Crossing fingers and continuing...", e);
+      }
+    }
+    
+    return false;
+    
+  }
+
   private boolean repairHasSegmentRunning(long repairRunId) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "repairHasSegmentRunning" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)